src/reporter/query_1TNE1A.py

```diff
@@ -83,6 +83,22 @@ def query_1TNE1A(attr_name,   # In Path
         logging.getLogger(__name__).error(msg, exc_info=True)
         return msg, 500
 
+    if err == "AggrMethod cannot be applied":
+        r = {
+            "error": "AggrMethod cannot be applied",
+            "description": "AggrMethod cannot be applied on type TEXT and BOOLEAN."
+        }
+        logging.getLogger(__name__).info("AggrMethod cannot be applied")
+        return r, 404
+
+    else:
```

Comment:
Remove the else here, it’s not needed

Reply:
By remove the else I mean the all code introduced with it. Because with this code you now always return 404 not found, even when there are results. Also 404 not found is handled below, when entities returned are none

Reply:
done